src/js/subtraction/components/List.jsx

```diff
@@ -1,58 +1,62 @@
 import { findSubtractions } from "../actions";
 import SubtractionItem from "./Item";
 import SubtractionToolbar from "./Toolbar";
 
-export class SubtractionList extends React.Component {
-    componentDidMount() {
-        if (!this.props.fetched) {
-            this.props.onLoadNextPage(this.props.term, 1);
+export const SubtractionList = ({ onLoadNextPage, documents, page, page_count, total_count, fetched, term }) => {
+    useEffect(() => {
+        if (!fetched) {
+            onLoadNextPage(term, 1);
         }
-    }
+    }, []);
 
-    renderRow = index => <SubtractionItem key={index} index={index} />;
+    const renderRow = index => <SubtractionItem key={index} index={index} />;
+    let subtractionComponents;
 
-    render() {
-        let subtractionComponents;
-        if (this.props.documents === null) {
-            return <LoadingPlaceholder />;
-        }
+    if (documents === null) {
+        return <LoadingPlaceholder />;
+    }
 
-        if (this.props.documents.length) {
-            subtractionComponents = (
-                <LegacyScrollList
-                    documents={this.props.documents}
-                    onLoadNextPage={page => this.props.onLoadNextPage(this.props.term, page)}
-                    page={this.props.page}
-                    pageCount={this.props.page_count}
-                    renderRow={this.renderRow}
-                />
-            );
-        } else {
-            subtractionComponents = <NoneFoundBox noun="subtractions" />;
-        }
+    if (documents.length) {
+        subtractionComponents = (
+            <LegacyScrollList
+                documents={documents}
+                onLoadNextPage={page => onLoadNextPage(term, page)}
+                page={page}
+                pageCount={page_count}
+                renderRow={renderRow}
+            />
+        );
+    } else {
+        subtractionComponents = <NoneFoundBox noun="subtractions" />;
+    }
 
-        return (
-            <>
-                <ViewHeader title="Subtractions">
-                    <ViewHeaderTitle>
-                        Subtractions <Badge>{this.props.total_count}</Badge>
-                    </ViewHeaderTitle>
-                </ViewHeader>
+    return (
+        <>
+            <ViewHeader title="Subtractions">
+                <ViewHeaderTitle>
+                    Subtractions <Badge>{total_count}</Badge>
+                </ViewHeaderTitle>
+            </ViewHeader>
 
-                <SubtractionToolbar />
+            <SubtractionToolbar />
 
-                {subtractionComponents}
-            </>
-        );
-    }
-}
+            {subtractionComponents}
+        </>
+    );
+};
 
 const mapStateToProps = state => ({
     ...state.subtraction,
+    fetched: state.subtraction.fetched,
+    documents: state.subtraction.documents,
+    page: state.subtraction.page,
+    page_count: state.subtraction.page_count,
+    total_count: state.subtraction.total_count,
+    term: state.subtraction.term,
```

Comment:
```suggestion
    fetched: state.subtraction.fetched,
    documents: state.subtraction.documents,
    page: state.subtraction.page,
    page_count: state.subtraction.page_count,
    total_count: state.subtraction.total_count,
    term: state.subtraction.term,
```
Generally a fan of reducing the usage of spread operators. That said you might be able to reduce the verbosity of this by taking advantage of object destructuring.